src/renderer/components/watch-video-chapters/watch-video-chapters.js

```diff
@@ -67,6 +71,24 @@ export default defineComponent({
       }
 
       chapterElements[newIndex].focus()
-    }
+    },
+
+    toggleShowChapters() {
+      this.showChapters = !this.showChapters
+
+      if (this.showChapters) { this.scrollToCurrentChapter() }
+    },
+
+    scrollToCurrentChapter() {
+      const container = this.$refs.chaptersWrapper
+      const currentChaptersItem = (this.$refs.currentChaptersItem || [])[0]
+      // Must wait until rendering done after value change
+      nextTick(() => {
+        if (container != null && currentChaptersItem != null) {
+          // Watch view can be ready sooner than this component
```

Comment:
This comment seems like a copy paste issue, as it doesn't seem relevant here (presumably this came from the playlist component).

Reply:
Removed